subo/command/se2_deploy.go

```diff
@@ -93,7 +96,9 @@ func SE2DeployCommand() *cobra.Command {
 				}
 
 				data := deployData{
+					SE2Tag:     tag,
 					SE2Version: tag,
```

Comment:
Prefer `XTag` here because they represent tags, not versions (which are similar but different because of prefixed v).